### DISCUSSION TITLE
What are the VSCode settings fixes Patrick did off-screen at 10:34:25?

### DISCUSSION THREAD

**Discussion by paprikrumplikas** [Cyfrin/security-and-auditing-full-course-s23#97] [Q&A]
Hi,

Can anyone share what VSCode settings fixes Patrick did off-screen at [https://youtu.be/pUWmJ86X_do?t=38065](url)?

After these modifications, the custom contract variable types like "MockUSDC" and "YeildERC20" got a nice light-blue coloring highlight, just as regular variable types like "address" and "uint256" have. 

Cheers,
paprik

**Comment by johnsonstephan** [comment]
You may be referring to syntax highlighting from the [Solidty VS Code extension ](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity). Also, change your theme to Dark. 

Did this work?

**Reply by paprikrumplikas** [reply]
Hi @johnsonstephan,

Thanks for your answer. 
Yes, I was referring to syntax highlighting. I do have the [Solidty VS Code extension](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) installed and enabled, yet custom contract variable types like "MockUSDC" and "YeildERC20" do not get  any syntax highlighting for me - and for Patrick either, that is why he did some fixes off-screen. 
I wonder what modification I need to do to achieve the same.

**Comment by EngrPips** [comment]
@paprikrumplikas  i think `Solidity Visual Developer` is the extension that gives those color highlighting.

**Reply by paprikrumplikas** [reply]
Thank you for the suggestion.
I have that one installed and enabled, yet custom contract variable types like "MockUSDC" and "YeildERC20" do not get any syntax highlighting for me. Patrick did not get any syntax highlighting for these either, that is why he did some off-screen modifications in some settings I cannot for the love of my life find.

**Comment by PatrickAlphaC** [comment]
I'm using the hardhat VSCode extension actually!

https://hardhat.org/hardhat-vscode/docs/overview

Try this one out!

**Reply by monaldmuck** [reply]
I have tried so many format settings still no color semantic changes, and I have hardhat enabled

**Reply by PatrickAlphaC** [reply]
Might be something wrong with you VSCode settings :/ 

It might be worth trying to debug with chatgpt or something.
